kratos/tests/cpp_tests/geometries/test_triangle_2d_6.cpp

```diff
@@ -74,7 +74,7 @@ namespace {
 
     KRATOS_TEST_CASE_IN_SUITE(Triangle2D6Volume, KratosCoreGeometriesFastSuite) {
         auto geom = GenerateReferenceTriangle2D6();
-        KRATOS_CHECK_EXCEPTION_IS_THROWN(geom->Volume(), "Calling base class 'Volume' method instead of derived class one.");
+        KRATOS_CHECK_NEAR(geom->Volume(), 0.0, TOLERANCE);
```

Comment:
what is the value of tolerance here? it should be very small

Reply:
It is defined on the beginning of the file, 1.0e-8

Reply:
i would make this much stricter.

Reply:
It would affect other tests